scripts/package-build/frr/package.toml

```diff
@@ -1,20 +1,20 @@
 [[packages]]
 name = "libyang"
-commit_id = "v2.1.148"
+commit_id = "v3.4.2"
 scm_url = "https://github.com/CESNET/libyang.git"
 build_cmd = "pipx run apkg build -i && find pkg/pkgs -type f -name *.deb -exec mv -t .. {} +"
 
 [[packages]]
 name = "rtrlib"
 commit_id = "v0.8.0"
 scm_url = "https://github.com/rtrlib/rtrlib.git"
-build_cmd = "sudo mk-build-deps --install --tool 'apt-get --yes --no-install-recommends'; dpkg-buildpackage -uc -us -tc -b"
+build_cmd = "dpkg-buildpackage -uc -us -tc -b"
```

Comment:
build_cmd could be omitted
https://github.com/vyos/vyos-build/blob/5bccbebc1eababfa856d82992b0cd593bbb37de9/scripts/package-build/build.py#L125

Reply:
Thanks - removed.